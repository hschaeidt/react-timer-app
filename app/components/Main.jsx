import React from 'react'
import Navigation from './Navigation'

export default (props) => (
  <div>
    <div>
      <div>
        <Navigation />
        <div className="column small-centered medium-6 large-4">
          {props.children}
        </div>
      </div>
    </div>
  </div>
)
