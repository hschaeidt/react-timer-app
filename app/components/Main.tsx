import * as React from 'react';
import Navigation from './Navigation';

const Main = (props: React.Props<any>) => (
  <div>
    <div>
      <div>
        <Navigation />
        <div className="column small-centered medium-6 large-4">
          {props.children}
        </div>
      </div>
    </div>
  </div>
);

export default Main;
