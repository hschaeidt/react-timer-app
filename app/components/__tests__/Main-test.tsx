import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Main from '../Main';

describe('<Main />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Main />);
    expect(tree).toMatchSnapshot();
  });
});
