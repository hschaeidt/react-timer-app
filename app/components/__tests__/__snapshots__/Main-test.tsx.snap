exports[`<Main /> renders correctly 1`] = `
<div>
  <div>
    <div>
      <div
        className="top-bar">
        <div
          className="top-bar-left">
          <ul
            className="menu">
            <li
              className="menu-text">
              Timer-App
            </li>
            <li>
              <a
                onClick={[Function]}
                style={Object {}}>
                Timer
              </a>
            </li>
            <li>
              <a
                onClick={[Function]}
                style={Object {}}>
                Countdown
              </a>
            </li>
            <li>
              <a
                onClick={[Function]}
                style={Object {}}>
                Tracker
              </a>
            </li>
          </ul>
        </div>
        <div
          className="top-bar-right">
          <ul
            className="menu">
            <li
              className="menu-text">
              Created by somebody
            </li>
          </ul>
        </div>
      </div>
      <div
        className="column small-centered medium-6 large-4" />
    </div>
  </div>
</div>
`;
